docs/reference/nuctl/nuctl.md

```diff
@@ -1,48 +1,63 @@
-# Nuctl
+# The Nuclio CLI (nuctl)
 
 #### In this document
 
-- [About](#about)
-- [Download](#download)
+- [Overview](#overview)
+- [Download and Installation](#download-n-install)
 - [Usage](#usage)
-- [Running platform](#running-platform)
-- [Docker](#docker)
-- [Kubernetes](#kubernetes)
+- [The running platform](#running-platform)
+  - [Docker](#docker)
+  - [Kubernetes](#kubernetes)
 
-## About
+<a id="overview"></a>
```

Comment:
is it really required? `<a></a>` isn't required when navigating within markdown, it is being added automatically.

Reply:
No it's not required, it's just good practice to set hardcoded anchors — it makes it simpler to link rather than deriving the auto-generated anchors, and more importantly, it frees us from the need to edit links to the headings if the title changes. That said, granted it's not really important for "Overview", I just like to use them everywhere for consistency and good measure and it doesn't hurt anyone :-).